### PR TITLE
Document new savedWidth and create migration guides

### PR DESCRIPTION
**Pull request by azirbel**
@Dylnuge @cyril-sf

### CODE REVIEW

**Comment by Dylnuge** [comment]
I don't know how I feel about retroactively updating changelogs, because the changelog will still be incorrect on the `0.3.0` release/tag, but otherwise LGTM

**Comment by azirbel** [comment]
On second thought, I share your concerns. I won't update the changelog for v0.3.0.

What I really want is a "migration guide" [here](http://addepar.github.io/ember-table/#/ember-table/migration-guides). I'll spend another few minutes and make that instead of messing with the changelog.

**Comment by azirbel** [comment]
@Dylnuge @cyril-sf @igillis I tried to write a more serious migration guide instead. See what you think =)

**Comment by Dylnuge** [comment]
LGTM.

File: app/templates/ember_table/migration-guides.hbs [hunk 1]
```diff
@@ -2,26 +2,122 @@
...
+          defaults to <code>false</code>, but if a column sets it to
+          <code>true</code>, the force fill behavior is implicitly triggered.
+        </li>
+      </ol>
+
+      <h2>Migrating to version 0.3.0</h2>
+      <p>
+        Version 0.3.0 includes major column resizing changes, but with
+        <code>canAutoResize</code> defaulting to <code>true</code>.  We
+        recommend skipping this version and upgrading direcly to 0.4.0; see
```
Comment (Dylnuge) [inline app/templates/ember_table/migration-guides.hbs]: directly

**Comment by cyril-sf** [comment]
Can you simply unregister the 0.3 release on a bower?

**Comment by azirbel** [comment]
It was short lived, we release 0.4.0 the next day and put "skip 0.3" in both the changelog and migration guides. I don't think we need to unregister it (it's a working version), it just makes upgrading more annoying if you do it step by step and don't read the guides.

**Comment by shaunc** [comment]
(IMHO unregistering w/ package managers is a bad idea. I recently got tied up in knots trying to install a package which included a sub-package w/ a version that had been deregistered in npm. The package owner couldn't see the problem because they already had installed locally.)

**Pull request merged**